lambdas/harvesters/ror/downloader.rb

```diff
@@ -0,0 +1,355 @@
+# frozen_string_literal: true
+
+# Docs say that the LambdaLayer gems are found mounted as /opt/ruby/gems but an inspection
+# of the $LOAD_PATH shows that only /opt/ruby/lib is available. So we add what we want here
+# and indicate exactly which folders contain the *.rb files
+my_gem_path = Dir['/opt/ruby/gems/**/lib/']
+$LOAD_PATH.unshift(*my_gem_path)
+
+require 'digest'
+require 'uri'
+require 'zip'
+
+require 'aws-sdk-s3'
+
+require 'uc3-dmp-api-core'
+require 'uc3-dmp-cloudwatch'
+require 'uc3-dmp-dynamo'
+require 'uc3-dmp-event-bridge'
+require 'uc3-dmp-external-api'
+
+module Functions
+  # A service that fetches the latest ROR file
```

Comment:
L22: I really liked the description for this downloader in the PR, and was wondering if you enhance this comment with some of those details: "It queries Zenodo for the latest ROR snapshot. It stashes the snapshot in S3, then unzips it and determines how many records there are. It then sends a message to an SNS topic for every 50,000 records. Those messages invoke the 2nd new Lambda"